tests/phpunit/TestFramework/PhpUnit/Coverage/IndexXmlCoverageParserTest.php

```diff
@@ -53,9 +55,6 @@
 
 /**
  * @group integration
- * @covers \Infection\TestFramework\PhpUnit\Coverage\IndexXmlCoverageParser
```

Comment:
We'll have to re-add this one line later if we'd want to practice what we preach about performance of MT.

Reply:
What do you mean? Remember we have the PHPUnit bridge listener which maps the `@covers`. It's not like we are on a free-well and it covers everything it's using.

Also each of those classes have their own dedicated test classes, so I don't think there is a need for this test class to cover them as well

Reply:
PHPUnit bridge listener does that for us? If so that's great.

Reply:
Yes: https://symfony.com/doc/current/components/phpunit_bridge.html#code-coverage-listener sorry I thought you were aware

Reply:
Beautiful, thanks. Shall we not have an autoreview test to guard against this?

Reply:
I briefly tried but gave up. It would be nice to have fore sure. Note that `@covers` is required in some cases though so we cannot just blindly remove it